### DISCUSSION TITLE
Nixos support ❄

### DISCUSSION THREAD

**Discussion by jlengelbrecht** [gravitational/teleport#48948] [Ideas]
What would you like Teleport to do?
I would to request support for nixos clients to be added to teleport. 

What problem does this solve?
Currently there is no support to add nixos clients to teleport for remote access

If a workaround exists, please include it.
No workaround to my knowledge

**Comment by zmb3** [comment]
This is not something we're planning on supporting at this time, but I'll convert this issue to a discussion and maybe you'll see some interest in the community.
